### DISCUSSION TITLE
Feature to add to albums more conviniently

### DISCUSSION THREAD

**Discussion by davidAeiou** [bpatrik/pigallery2#763] [Ideas]
Hello, 
Thank you very much for this great project, it is very useful in our home for viewing our family photo gallery. I appreciate your work! The folder structure as a base is exactly what I needed, not to mention the clean view, advanced settings, etc... 
I want to request a feature which I have in mind - regarding Albums.
Have you thought about a way to create the albums based on selection of direct photos by the user? Some button near the photo like Add to album - album name.
It would be maybe more useful than just the search query which creates the contents of the album.
I have certain use case regarding this - apart of creating proper albums, of course, I would use it to mark and group photos to delete later on directly in the file system. It happens that photos not needed are discovered while viewing and this would be nice just to add it to 1 album as kind of reminder and once I find time for it, I would delete/move them in the filesystem in a batch.

Thank you.

**Comment by kagahd** [comment]
Your suggestion sounds like this [feature request](https://github.com/bpatrik/pigallery2/issues/301) https://github.com/bpatrik/pigallery2/issues/301

**Comment by bpatrik** [comment]
I'm closing this discussion in favor of #301
